src/modeler/AutoRest.Swagger/Validation/Core/RecursiveObjectValidator.cs

```diff
@@ -116,7 +137,7 @@ private IEnumerable<LogMessage> RecursiveValidate(object entity, ObjectPath enti
             return classRules.SelectMany(rule => rule.GetValidationMessages(entity, parentContext));
         }
 
-        private IEnumerable<LogMessage> ValidateProperty(PropertyInfo prop, object value, ObjectPath entityPath, RuleContext parentContext)
+        private IEnumerable<LogMessage> ValidateProperty(PropertyInfo prop, object value, ObjectPath entityPath, RuleContext parentContext, ServiceDefinitionMetadata metaData)
```

Comment:
minor: add documentation (Personally, I'd be more inclined to have detailed docs when ever possible as that saves me for bugging people :) )